identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```
@@ -308,6 +308,7 @@ go.to.sign.up=Go to sign up
 scan.the.qr.code.using.an.authenticator.app=Scan the QR code using an authenticator app
 enter=Enter
 invalid.organization.name=Invalid Organization Name
+invalid.organization.handle=Invalid Organization Handle
```

shall we keep this error message in sentence case?

```suggestion
invalid.organization.handle=Invalid organization handle
```

---

This was added to match the existing **invalid.organization.name** error message.